### ISSUE TITLE
Optimize notification of the sync service about imported transactions

### ISSUE THREAD

**Issue by rphmeier** [paritytech/parity#8082]
https://github.com/paritytech/parity/pull/8074/files/1f1694b444592857bfd9811e72d4e40c0d835b64#r173615792

**Comment by folsen** [comment]
@tomusdrw Can this be closed after #8074?

**Comment by tomusdrw** [comment]
Not really, the link actually leads to a comment in that particular PR, @0x7CFE thougts? Is it worth implementiing in your opinion?

**Comment by 0x7CFE** [comment]
@tomusdrw, probably not. I wrote my opinion on that in [the trailing comment](https://github.com/paritytech/parity/pull/8074/files/1f1694b444592857bfd9811e72d4e40c0d835b64#r174054076).
